src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1343,10 +1354,11 @@ static bool processOnChangeConditionForUpdateContext
   const AttributeList&             attrL,
   std::string                      subId,
   std::string                      notifyUrl,
-  NotificationFormat               notifyFormat,
+  RenderFormat                     renderFormat,
   std::string                      tenant,
   const std::string&               xauthToken,
-  const std::string&               fiwareCorrelator
+  const std::string&               fiwareCorrelator,
+  std::vector<std::string>         attributeV
```

Comment:
The type doesn't conform the in/out criteria. I mean, it should be either

const std::vector<std::string> -> for "read only" parameters

or

std::vector<std::string>*   -> otherwise.

In addition, it this attribute is intended for define the attribute order, maybe `attrsOrderV` (or something like that) would be a better name.